src/main/resources/outcomeCodeLookup.txt

```diff
@@ -49,17 +49,17 @@
 03-02-07|NEW_ADDRESS_REPORTED
 03-02-08|NEW_ADDRESS_REPORTED,FULFILMENT_REQUESTED,LINKED_QID
 6-10-01|ADDRESS_TYPE_CHANGED_CE_EST,FULFILMENT_REQUESTED,LINKED_QID
-6-10-02|ADDRESS_NOT_VALID,HARD_REFUSAL_RECEIVED,FULFILMENT_REQUESTED,LINKED_QID
+6-10-02|ADDRESS_NOT_VALID,FULFILMENT_REQUESTED,LINKED_QID,HARD_REFUSAL_RECEIVED
```

Comment:
Is there a need to push these to the end of the list?

Reply:
Yes, just an extended delay. Current order is a mandatory order or its ok to change in process of outcome?